Application Actions/Preview/1.0/default.json

```diff
@@ -0,0 +1,9 @@
+{
+  "RotateLeft"  : "Scroll Down",
+  "RotateRight" : "Scroll Up",
+  "ButtonPress" : "Toggle Full Screen",
+  "SwipeUp"     : "Home",
+  "SwipeDown"   : "End",
+  "SwipeLeft"   : "Page Up",
+  "SwipeRight"  : "Page Down"
+}
```

Comment:
Sure - happy to change that. I wasn't super happy with the names anyway, but couldn't think of what to put. But perhaps without the "1" in the page up/down...? I'll go for that.